train.py

```diff
@@ -508,24 +546,24 @@ def main(args_not_parsed):
         g_optim.load_state_dict(ckpt["g_optim"])
         d_optim.load_state_dict(ckpt["d_optim"])
 
-    motion_data_raw = np.load(args.path, allow_pickle=True)
-    motion_data, mean_joints, std_joints, edge_rot_dict_general = motion_from_raw(args, motion_data_raw)
-
-    gt_bone_lengths = calc_bone_lengths(motion_data) if args.entity == 'Joint' else None
-
-    motion_path = osp.join(animations_output_folder, 'real_motion.bvh')
-    motion2bvh(motion_data[0], motion_path, parents=entity.parents_list, entity=args.entity,
-               edge_rot_dict_general=edge_rot_dict_general)
-    if args.clearml:
-        logger.report_media(title='Animation', series='Ground Truth Motion', iteration=0, local_path=motion_path)
-
-    fig = motion2fig(motion_data, H=512, W=512, entity=args.entity,
-                     edge_rot_dict_general=edge_rot_dict_general)
-    fig_name = osp.join(images_output_folder, 'real_motion.png')
-    fig.savefig(fig_name, dpi=300, bbox_inches='tight')
-    plt.close() # close figure
-    if args.clearml:
-        logger.report_media(title='Image', series='Ground Truth Motion', iteration=0, local_path=fig_name)
+    motion_data, mean_joints, std_joints, = motion_from_raw(args, motion_data_raw, static)
+
+    # Just save some real motion for start - TODO: Why?
```

Comment:
We would like to see what our GT data looks like. Please restore this commented block.